Lab/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Lab
+{
+     public class Program
+    {
+        public static double balance = 0;
+        public static void Main(string[] args)
+        {
+            Console.WriteLine(ViewBalance(balance));
+        }
+
+        public static double ViewBalance(double balance)
```

Comment:
I would just remove this parameter so you can return the balance *field* value instead, essentially

```js
var balance = 0;

function viewBalance() {
  return balance;
}
```